modules/common/src/main/java/org/opencastproject/mediapackage/MediaPackageSupport.java

```diff
@@ -539,21 +540,21 @@ public String apply(MediaPackage mp) {
     public static final Function<MediaPackage, List<MediaPackageElement>> getElements = new Function<MediaPackage, List<MediaPackageElement>>() {
       @Override
       public List<MediaPackageElement> apply(MediaPackage a) {
-        return Immutables.list(a.getElements());
+        return Arrays.asList(a.getElements());
```

Comment:
Very strictly speaking, you're losing something here, since `Arrays.asList` returns a mutable `ArrayList`. But I suppose `Collections.unmodifiableList(Arrays.asList(...))` is as ugly as it gets, so your solution fine with me. :)